runtime/src/main/java/org/corfudb/runtime/view/replication/QuorumReplicationProtocol.java

```diff
@@ -131,6 +132,10 @@ public void write(Layout layout, ILogData data) throws OverwriteException {
         }
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public void writeForwardpointer(Layout layout, UUID stream, long address, long nextAddress) {
```

Comment:
Quorum raises the question of how much coordination is needed to get the pointers replicated (are we just making copies or running paxos like the data replication)?